commit 1be50be5ebc5b038038cf9abb8fd3e52a24d03c1
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Wed Mar 29 14:28:53 2017 +0100

    config: Re-enable the bcm2835-mmc driver
    
    With the patch to assign mmc device IDs based on DT aliases and
    appropriate aliases in the rpi DTBs, it is now safe to re-enable
    the bcm2835-mmc driver.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index 8198d4f73e23..8163d3cf9303 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -1046,6 +1046,7 @@ CONFIG_USB_XUSBATM=m
 CONFIG_USB_GADGET=m
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
+CONFIG_MMC_BCM2835=y
 CONFIG_MMC_BCM2835_SDHOST=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 9eb7084f440c..021c38a909e7 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -1053,6 +1053,7 @@ CONFIG_USB_G_HID=m
 CONFIG_USB_G_WEBCAM=m
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
+CONFIG_MMC_BCM2835=y
 CONFIG_MMC_BCM2835_SDHOST=y
 CONFIG_MMC_SDHCI=y
 CONFIG_MMC_SDHCI_PLTFM=y
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1126_1be50be5ebc5b038038cf9abb8fd3e52a24d03c1.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1126_1be50be5ebc5b038038cf9abb8fd3e52a24d03c1.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
